matlab/library/WBToolboxLibrary_repository.mdl

```diff
@@ -1,7 +1,7 @@
 Library {
   Name			  "WBToolboxLibrary_repository"
   Version		  9.1
-  SavedCharacterEncoding  "US-ASCII"
+  SavedCharacterEncoding  "UTF-8"
```

Comment:
Clearly this is the ouput of an automatically generated file, but just checking @diegoferigo, to you think it is ok?

Reply:
I was wondering the same when I saw the diff. I guess this is related to the differences in the locale of Yeshi's and my machine. I don't think it's going to be a problem, but cannot guarantee. Let's keep this in mind.

Reply:
Resolving this conversation
@traversaro @diegoferigo